code/modules/mob/living/carbon/xenomorph/xeno_verbs.dm

```diff
@@ -25,6 +25,13 @@
 	if(!hive)
 		return
 
+	if(hive.hivenumber == XENO_HIVE_RENEGADE) //Renegade's ability to attack someone depends on IFF settings, not on alliance
+		if(!iff_tag)
+			to_chat(src, SPAN_NOTICE("You are not oblidged to protect anyone."))
```

Comment:
```suggestion
			to_chat(src, SPAN_NOTICE("You are not obligated to protect anyone."))
```
obliged can work too, but its not a very common usage of the word thus it being consider archaic: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/fc9f3c99-4224-436e-a5e1-03ba1955f8e9)